lib/src/interface/common.dart

```diff
@@ -26,7 +26,7 @@ String sanitizeExecutablePath(String executable,
   if (!platform.isWindows) {
     return executable;
   }
-  if (executable.contains(' ') && !executable.startsWith('"')) {
+  if (executable.contains(' ') && !executable.contains('"')) {
```

Comment:
What if it contains `\ ` (an escaped space)?

Reply:
I could be wrong, but my understanding is that:` C:\"foo"\bar` resolves as `C:\foo\bar`.  If this is true, then ` C:\"foo^ bar"` resolves as  ` C:\foo^ bar`.  (`^ ` is Windows way to escape spaces). I don't have a Windows computer, but let me ask to see if we can verify it.

Reply:
These paths are resolved correctly:
```.dart
Process.runSync('"C:\\Users\\franciscojma\\Hello World\\git.exe"', const <String>[]);
Process.runSync('\"C:\\Users\\franciscojma\\Hello World\\git.exe\"', const <String>[]);
```

The ones below returned "Process not found":
```.dart
Process.runSync('C:\\Users\\franciscojma\\"Hello World"\\git.exe', const <String>[]);
Process.runSync('C:\\Users\\franciscojma\\"Hello^ World"\\git.exe', const <String>[]);
Process.runSync('"C:\\Users\\franciscojma\\Hello^ World"\\git.exe"', const <String>[]);
Process.runSync('C:\\Users\\franciscojma\\Hello^ World\\git.exe', const <String>[]);
```